frontend/config/flags.ts

```diff
@@ -0,0 +1,4 @@
+export const flags = {
+   PRODUCT_PAGE_ENABLED:
+      process.env.NEXT_PUBLIC_FLAG__PRODUCT_PAGE_ENABLED === 'true',
```

Comment:
Is this the same as saying?

```
	PRODUCT_PAGE_ENABLED: process.env.NEXT_PUBLIC_FLAG__PRODUCT_PAGE_ENABLED ?? false
```

Reply:
Ahh scratch that, I see that `process.env.NEXT_PUBLIC_FLAG__PRODUCT_PAGE_ENABLED` can be a string so `'false'` would return true.